packages/ccda/src/types.ts

```diff
@@ -253,16 +253,16 @@ export type CcdaValue = CcdaCode | CcdaText | CcdaQuantity | CcdaReference;
 
 export interface CcdaPeriod {
   '@_xsi:type'?: 'PIVL_TS';
-  '@_value': string;
-  '@_unit': string;
+  '@_value'?: string;
+  '@_unit'?: 's' | 'min' | 'h' | 'd' | 'wk' | 'mo' | 'a';
 }
 
 export interface CcdaEvent {
   '@_code': string;
 }
 
 export interface CcdaEffectiveTime {
-  '@_xsi:type'?: 'IVL_TS' | 'TS';
+  '@_xsi:type'?: 'IVL_TS' | 'TS' | 'PIVL_TS';
```

Comment:
Interesting

Reply:
For reference:

https://build.fhir.org/ig/HL7/CDA-core-2.0/StructureDefinition-PIVL-TS-definitions.html
https://build.fhir.org/ig/HL7/CDA-core-2.0/StructureDefinition-IVL-TS.html

Reply:
`xsi:type` is indeed quite "interesting"